python/ray/data/_internal/execution/interfaces/op_runtime_metrics.py

```diff
@@ -376,37 +456,39 @@ class OpRuntimeMetrics(metaclass=OpRuntimesMetricsMeta):
         description="Time spent in task output backpressure.",
         metrics_group=MetricsGroup.TASKS,
     )
-    histogram_buckets_s = [
-        0.1,
-        0.25,
-        0.5,
-        1.0,
-        2.5,
-        5.0,
-        7.5,
-        10.0,
-        15.0,
-        20.0,
-        25.0,
-        50.0,
-        75.0,
-        100.0,
-        150.0,
-        500.0,
-        1000.0,
-        2500.0,
-        5000.0,
-    ]
-    task_completion_time: float = metric_field(
-        default=0,
+    task_completion_time: list[int] = metric_field(
+        default_factory=list,
         description="Time spent running tasks to completion.",
         metrics_group=MetricsGroup.TASKS,
+        metrics_type=MetricsType.Histogram,
+        metrics_args={"boundaries": histogram_buckets_s},
+    )
+    block_completion_time: list[int] = metric_field(
+        default_factory=list,
+        description="Time spent running a single block to completion. If multiple blocks are generated per task, this is approximated by assuming each block took an equal amount of time to process.",
+        metrics_group=MetricsGroup.TASKS,
+        metrics_type=MetricsType.Histogram,
+        metrics_args={"boundaries": histogram_buckets_s},
     )
     task_completion_time_without_backpressure: float = metric_field(
```

Comment:
@iamjustinhsu ,

should i delete this metric now? Is it still useful if we have the histograms now?